homeassistant/components/otbr/__init__.py

```diff
@@ -2,29 +2,56 @@
 from __future__ import annotations
 
 import dataclasses
+from functools import wraps
 
 import python_otbr_api
 
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import HomeAssistantError
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.typing import ConfigType
 
+from . import websocket_api
 from .const import DOMAIN
 
 
+def _handle_otbr_error(func):
+    """Handle OTBR errors."""
+
+    @wraps(func)
+    async def _func(self, *args, **kwargs):
```

Comment:
Side note: It seems this integration isn't strictly typed. Would be nice to do that asap so it's not a hurdle later. And preferably the library should also be typed.